scripts/testnet/include/config.sh

```diff
@@ -0,0 +1,152 @@
+source "$ELRONDTESTNETSCRIPTSDIR/variables.sh"
+
+generateConfig() {
+  echo "Generating configuration using values from scripts/variables.sh..."
+
+  cd $TESTNETDIR/filegen
+  ./filegen \
+    -mint-value $MINT_VALUE                               \
+    -num-of-shards $SHARDCOUNT                            \
+    -num-of-nodes-in-each-shard $SHARD_VALIDATORCOUNT     \
+    -num-of-observers-in-each-shard $SHARD_OBSERVERCOUNT  \
+    -consensus-group-size $SHARD_CONSENSUS_SIZE           \
+    -num-of-metachain-nodes $META_VALIDATORCOUNT          \
+    -num-of-observers-in-metachain $META_OBSERVERCOUNT    \
+    -metachain-consensus-group-size $META_CONSENSUS_SIZE  \
+    -consensus-type $CONSENSUS_TYPE
+}
+
+copyConfig() {
+  cd $TESTNETDIR
+
+  cp ./filegen/genesis.json ./node/config
+  cp ./filegen/nodesSetup.json ./node/config
+  cp ./filegen/initialBalancesPkPlain.txt ./node/config
+  cp ./filegen/initialBalancesSk.pem ./node/config
+  cp ./filegen/initialBalancesSkPlain.txt ./node/config
+  cp ./filegen/initialNodesPkPlain.txt ./node/config
+  cp ./filegen/initialNodesSk.pem ./node/config
+  cp ./filegen/initialNodesSkPlain.txt ./node/config
+  echo "Configuration files copied from the configuration generator to the working directories of the executables."
+}
+
+copySeednodeConfig() {
+  cd $TESTNETDIR
+  cp $SEEDNODEDIR/config/p2p.toml ./seednode/config
+}
+
+updateSeednodeConfig() {
+  cd $TESTNETDIR/seednode/config
+  cp p2p.toml p2p_edit.toml
+
+  updateTOMLValue p2p_edit.toml "Port" $PORT_SEEDNODE
+
+  cp p2p_edit.toml p2p.toml
+  rm p2p_edit.toml
+
+  echo "Updated configuration for the Seednode."
+}
+
+copyNodeConfig() {
+  cd $TESTNETDIR
+  cp $NODEDIR/config/config.toml ./node/config
+  cp $NODEDIR/config/economics.toml ./node/config
+  cp $NODEDIR/config/prefs.toml ./node/config
+  cp $NODEDIR/config/server.toml ./node/config
+  cp $NODEDIR/config/p2p.toml ./node/config
+
+  echo "Configuration files copied from the Node to the working directories of the executables."
+}
+
+updateNodeConfig() {
+  cd $TESTNETDIR/node/config
+  cp p2p.toml p2p_edit.toml
+
+  updateTOMLValue p2p_edit.toml "InitialPeerList" "[\"$P2P_SEEDNODE_ADDRESS\"]"
+  cp p2p_edit.toml p2p.toml
+  rm p2p_edit.toml
+
+  echo "Updated configuration for Nodes."
+}
+
+copyProxyConfig() {
+  cd $TESTNETDIR
+
+  cp $PROXYDIR/config/config.toml ./proxy/config/
+
```

Comment:
maybe do updateProxyConfig at this point

Reply:
Fair enough - I'll keep `copyProxyConfig` and `updateProxyConfig` separate for now, so it will be visible in `start.sh` that the config files are in fact being edited. Same applies for the other config functions.